lib/box-node-sdk.js

```diff
@@ -273,20 +273,22 @@ BoxSDKNode.prototype.getTokensAuthorizationCodeGrant = function(authorizationCod
  * Refreshes the access and refresh tokens for a given refresh token.
  *
  * @param {string} refreshToken - A valid OAuth refresh token
+ * @param {?TokenRequestOptions} options - Sets optional behavior for the token grant, null for default behavior
  * @param {Function} callback - passed a TokenInfo object if tokens were granted successfully
  * @returns {void}
  */
-BoxSDKNode.prototype.getTokensRefreshGrant = function(refreshToken, callback) {
-	this.tokenManager.getTokensRefreshGrant(refreshToken, callback);
+BoxSDKNode.prototype.getTokensRefreshGrant = function(refreshToken, options, callback) {
```

Comment:
These SDK interfaces are public; I believe they need to be made backwards-compatible via argument shuffling